pkg/virt-handler/vm.go

```diff
@@ -1432,6 +1461,12 @@ func (c *VirtualMachineController) Run(threadiness int, stopCh chan struct{}) {
 	defer c.Queue.ShutDown()
 	log.Log.Info("Starting virt-handler controller.")
 
+	err := findMigrationIP(&c.migrationIpAddress)
```

Comment:
Would it make sense to look the ip address up outside of the controller and pass it as the desired IP address instead of implicitly overriding the passed in IP address ad creation time? The rename to `migrationIPAddress` is great if that is really the only reason why we ever use it.

Reply:
Done!